### PR TITLE
Docker cherry pick

### PR DESCRIPTION
**Pull request by heinrichmartin**
These are the changes from the docker branch that have an impact on the legacy code:
* rename `sql/create*` to have them in lexical order of the README.md
* `coceso.properties.default`: empty `authUrl` crashes the app
* `coceso.properties.default`: have the geocode table in the same database by default
* `**/pom.xml`: allow to easily exclude radio plugin (defaults to legacy behavior, exclude with `-P -radio`)

### CODE REVIEW

File: main/view/src/main/webapp/WEB-INF/classes/coceso.properties.default [hunk 1]
```diff
@@ -20,7 +20,7 @@ js.plugins={}
 
 # Geocode DB config
 geocode.db.driver=org.postgresql.Driver
-geocode.db.url=jdbc:postgresql://localhost:5432/geocode
+geocode.db.url=jdbc:postgresql://localhost:5432/coceso
```
Comment (robo-w) [inline main/view/src/main/webapp/WEB-INF/classes/coceso.properties.default]: in the default setting the coceso backend and the geocode addon write to different databases. therefore the different example was mentioned here.

Comment (heinrichmartin) [reply main/view/src/main/webapp/WEB-INF/classes/coceso.properties.default]: Shall I undo this change (and fix the README.md to be clear about the layout of the databases)?

Comment (robo-w) [reply main/view/src/main/webapp/WEB-INF/classes/coceso.properties.default]: yes please

**Comment by heinrichmartin** [comment]
Keeping the branch for of the same; will merge `docker` only when it runs smoothly.

**Pull request merged**